tests/TokenTest.py

```diff
@@ -23,15 +23,26 @@ def test_line_tokenizer(self):
 
     def test_string_tokenizer(self):
         text = '20. u2-sza-bi-la-kum\n1. a-na ia-as2-ma-ah-{d}iszkur#\n2. qi2-bi2-ma\n3. um-ma {d}utu-szi-{d}iszkur\n4. a-bu-ka-a-ma\n5. t,up-pa-[ka] sza#-[tu]-sza-bi-lam esz-me\n6. asz-szum t,e4#-em# {d}utu-illat-su2\n7. u3 ia#-szu-ub-dingir sza a-na la i-[zu]-zi-im\n'
-        output = Tokenizer.string_line_tokenizer(text)
+        output = Tokenizer.string_tokenizer(text)
         goal = ['20. u2-sza-bi-la-kum', '1. a-na ia-as2-ma-ah-{d}iszkur#', '2. qi2-bi2-ma', 
         '3. um-ma {d}utu-szi-{d}iszkur', '4. a-bu-ka-a-ma', '5. t,up-pa-[ka] sza#-[tu]-sza-bi-lam esz-me', 
         '6. asz-szum t,e4#-em# {d}utu-illat-su2', '7. u3 ia#-szu-ub-dingir sza a-na la i-[zu]-zi-im']
         self.maxDiff = None
         self.assertEqual(output, goal)
 
     def test_word_tokenizer(self):
+        text = r"C:\\Users\\andrew.deloucas\\GSoC-Akkadian\\texts\\ARM1Akkadian.txt"
+        lines = Tokenizer.line_tokenizer(text)
```

Comment:
I wouldn't use the output from the line_tokenizer here, as then the test will fail if the line_tokenizer is broken even when the word_tokenizer still works.  Give some pre-processed text here to input into the word_tokenizer.